### ISSUE TITLE
Unable to build the project - Angular errors in the console

### ISSUE THREAD

**Issue by cpavnn** [forcedotcom/IoraLite#6]
After installing the front end and server dependencies and when I run `npm start`, I get the following errors
I suspect something with the version of Angular JS
`
Uncaught TypeError: angular.module(...).info is not a function
    at angular-route.js:104
    at angular-route.js:1266
`

![image](https://user-images.githubusercontent.com/12685087/75545205-b097b200-5a4b-11ea-9734-a885f966435d.png)

**Comment by sfdeveloperhub** [comment]
@cpavnn are you still facing the issue?

**Comment by cpavnn** [comment]
Yes @sfdeveloperhub  I tried for couple of hours, couldnt get this working

**Comment by KaustubhBahl** [comment]
To fix the issue, install versions of angular-route and angular-resource same as angular. Angular version can be found in the bower.json file under bower_components/angular.
To install a specific version of the angular-route and angular-resource:
`bower install angular-route#<version_number>`
`bower install angular-resource#<version_number>`
